docs/install/1-install-server.rst

```diff
@@ -357,8 +357,17 @@ to
   # IPv6 local connections:
   host    all             all             ::1/128                 md5
 
+Make sur JIT is disabled for the psql database as it will impact performances.
+To disable JIT, set
 
-Restart the PostgreSQL server to apply the changes made in the ``pg_hba.conf`` file:
+.. code-block:: text
+
+  # disable JIT
+  jit = off
+
+in ``/etc/postgresql/13/main/postgresql.conf``.
+
+Restart the PostgreSQL server to apply the changes made in the ``pg_hba.conf`` and  ``postgresql.conf`` files:
```

Comment:
`pg_hba.conf`?

Reply:
I don't know what this file is it was already in the docs I didn't add it

Reply:
Ah. I see. This is a section with all the PostgreSQL-related config and not only the GIT config.